lesson4/script.js

```diff
@@ -0,0 +1,6 @@
+function multiplier() {
+	let inputNumber1 = prompt("Enter  a number");
+	let inputNumber2 = prompt("Enter  another number");
+	let product = inputNumber1 * inputNumber2;
```

Comment:
I'd recommend using `Number()` just to be super clear that the inputs are numbers, not strings.